packages/components/README.md

```diff
@@ -6,56 +6,73 @@ Reusable UI components for Deriv.
 
 **In this document**
 
--   [Other documents](#other-documents)
 -   [Commands](#commands)
 -   [Storybook](#storybook)
 -   [Usage](#usage)
 
-## Other documents
-
--   [General](docs/README.md) - Contains general philosophy and overview of this package
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
-
 ## Commands
 
 Setup:
+The `npm run bootstrap` command should be run in the root of the project, because it does not exist in the components package.js
 
 ```sh
-lerna bootstrap
+npm run bootstrap
 ```
 
-Serve:
+Build:
+The `npm run build` command is used to build the components package
 
 ```sh
-npm run serve
+npm run build
 ```
 
-Build:
+Serve:
+The `npm run serve` command is used to serve the components package
 
 ```sh
-npm run build
+npm run serve components
 ```
 
 Test:
+The `npm run test` command is used to check whether the icon names are valid in the components package
```

Comment:
when we run this command it automatically runs these 3 commands: `npm run test:stylelint`  && `npm run test:eslint` && `npm run test:jest` 
as far as I know it runs all the tests and make sure that every tests pass. I think it's better to double check this with @ali-hosseini-fs or one of the tech leads

Reply:
I actually tagged them to review this, but as you can see in the attached screenshots, this does not require double checking as it is clear in the commands themselves. You should only consider the components package.json here and not the root package.json

<img width="1664" alt="root package json" src="https://user-images.githubusercontent.com/104334373/180338418-cdbd5946-836d-46ea-a4fb-8b526fc89ed4.png">
<img width="1657" alt="components package json" src="https://user-images.githubusercontent.com/104334373/180338453-d0899e0d-1327-471b-9cdf-8278bafabe59.png">

Reply:
this command does more than just check the icon names 
```The `npm run test` command is used to check whether the icon names are valid in the components package```
is it right?

Reply:
Hello @ali-hosseini-fs,

Below is the code, maybe I deciphered  it wrong, I need you to help me decipher it as well.

<img width="1179" alt="icons" src="https://user-images.githubusercontent.com/104334373/180435269-c3311d73-5957-4e51-9bc5-040f5e8d5dec.png">

Thank you

Reply:
You are right Jim, I tried that one more time and it is just testing the icons.

Reply:
Thank you @niloo-fs